src/Microsoft.ML.Data/Evaluators/EvaluatorUtils.cs

```diff
@@ -44,7 +44,7 @@ private static class DefaultEvaluatorTable
                             { MetadataUtils.Const.ScoreColumnKind.Regression, env => new RegressionMamlEvaluator(env, new RegressionMamlEvaluator.Arguments()) },
                             { MetadataUtils.Const.ScoreColumnKind.MultiOutputRegression, env => new MultiOutputRegressionMamlEvaluator(env, new MultiOutputRegressionMamlEvaluator.Arguments()) },
                             { MetadataUtils.Const.ScoreColumnKind.QuantileRegression, env => new QuantileRegressionMamlEvaluator(env, new QuantileRegressionMamlEvaluator.Arguments()) },
-                            { MetadataUtils.Const.ScoreColumnKind.Ranking, env => new RankerMamlEvaluator(env, new RankerMamlEvaluator.Arguments()) },
+                            { MetadataUtils.Const.ScoreColumnKind.Ranking, env => new RankingMamlEvaluator(env, new RankingMamlEvaluator.Arguments()) },
```

Comment:
>RankingMamlEvaluator [](start = 86, length = 20)

Note that this renaming is correct. You are not evaluating a ranker, you are evaluating a ranking, so this one is good.

Reply:
It is also you note now *more* consistent with its "brethren" here.

---
In reply to: [258754361](https://github.com/dotnet/machinelearning/pull/2675#discussion_r258754361) [](ancestors = 258754361)